Firmware/MotorControl/low_level.c

```diff
@@ -1224,6 +1273,18 @@ static void control_motor_loop(Motor_t* motor) {
 
         // Velocity control
         float Iq = motor->current_setpoint;
+
+        // Anti-cogging is enabled after calibration
+        // We get the current position and apply a current feed-forward
+        // ensuring that we handle negative encoder positions properly (-1 == ENCODER_CPR - 1)
+        if(motor->anticogging.use_anticogging){
+            int32_t cogPos = (int32_t)motor->encoder.pll_pos % (int32_t)ENCODER_CPR;
+            if(cogPos < 0){
+                cogPos += ENCODER_CPR;
+            }
```

Comment:
I went through the corner cases in this calculation in my head and verified it should work, provided we don't overflow int32.